src/test/java/org/entando/plugins/pda/core/service/task/FakeTaskService.java

```diff
@@ -35,13 +35,12 @@
     public static final String TASK_PROCESS_INSTANCE_ID_2 = "processInstanceId2";
```

Comment:
Most of the constants in this class are not being used. If they are not going to be used, it is better to remove them.